packages/rum-core/src/domain/rumEventsCollection/error/trackReportError.spec.ts

```diff
@@ -0,0 +1,45 @@
+import type { RawError, Subscription } from '@datadog/browser-core'
+import { ErrorHandling, ErrorSource, Observable, clocksNow, isChromium } from '@datadog/browser-core'
+import type { Clock } from '../../../../../core/test/specHelper'
+import { stubReportingObserver, mockClock } from '../../../../../core/test/specHelper'
+import { trackReportError } from './trackReportError'
+
+describe('trackReportError', () => {
+  let errorObservable: Observable<RawError>
+  let subscription: Subscription
+  let notifyLog: jasmine.Spy
+  let clock: Clock
+  let reportingObserverStub: { raiseReport(type: string): void; reset(): void }
+
+  beforeEach(() => {
+    errorObservable = new Observable()
+    notifyLog = jasmine.createSpy('notifyLog')
+    reportingObserverStub = stubReportingObserver()
+    trackReportError(errorObservable)
+    subscription = errorObservable.subscribe(notifyLog)
+    clock = mockClock()
+  })
+
+  afterEach(() => {
+    subscription.unsubscribe()
+    clock.cleanup()
+    reportingObserverStub.reset()
+  })
+
+  it('should error report', () => {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏should it not be enable only behind the FF?

Reply:
Of course